mathesar_ui/src/systems/table-view/table-inspector/table/TableMode.svelte

```diff
@@ -1,47 +1,55 @@
 <script>
   import { Collapsible } from '@mathesar-component-library';
-  import TableConstraints from '@mathesar/systems/table-view/constraints/TableConstraints.svelte';
   import RenameTable from './RenameTable.svelte';
-  import ConstraintHelp from '../../constraints/__help__/ConstraintHelp.svelte';
   import TableActions from './TableActions.svelte';
+  import CollapsibleHeader from '../CollapsibleHeader.svelte';
+  import AdvancedActions from './AdvancedActions.svelte';
+  import TableLinks from './links/TableLinks.svelte';
 </script>
 
 <div class="table-mode-container">
   <Collapsible isOpen>
-    <span slot="header">Table Properties</span>
-    <div slot="content" class="property-container">
+    <CollapsibleHeader
+      slot="header"
+      title="Properties"
+      isDBLevelConfiguration
+    />
+    <div slot="content" class="content-container">
       <RenameTable />
     </div>
```

Comment:
The input to set description is missing here.

Reply:
We do support table descriptions. It's implemented on the backend.

Reply:
@rajatvijay Please create a separate issue for this if you're planning to handle this in a different PR.

Reply:
I created https://github.com/centerofci/mathesar/issues/1990 for this